wolfcrypt/src/memory.c

```diff
@@ -1660,6 +1660,39 @@ void __attribute__((no_instrument_function))
 }
 #endif
 
+#ifndef WOLFSSL_NO_FORCE_ZERO
+/* Exported version of ForceZero() that takes a size_t. */
+void wc_ForceZero(void *mem, size_t len)
+{
+    byte *zb = (byte *)mem;
+    unsigned long *zl;
+
+    XFENCE();
+
+    while ((unsigned long)zb & (unsigned long)(sizeof(unsigned long) - 1U)) {
+        if (len == 0)
```

Comment:
Decrementing len?

Reply:
d'oh!

Windows was giving a warning on the cast from `void *` to `unsigned long` so I had to push changes anyway, but, d'oh!